CMakeLists.txt

```diff
@@ -341,6 +341,23 @@ set (CMAKE_ASM_FLAGS                     "${CMAKE_ASM_FLAGS} ${COMPILER_FLAGS} $
 set (CMAKE_ASM_FLAGS_RELWITHDEBINFO      "${CMAKE_ASM_FLAGS_RELWITHDEBINFO} -O3 ${DEBUG_INFO_FLAGS} ${CMAKE_ASM_FLAGS_ADD}")
 set (CMAKE_ASM_FLAGS_DEBUG               "${CMAKE_ASM_FLAGS_DEBUG} -O0 ${DEBUG_INFO_FLAGS} ${CMAKE_ASM_FLAGS_ADD}")
 
+# Disable frame pointer compiler optimization using -fno-omit-frame-pointer
```

Comment:
maybe add a comment that it is needed for `fp` method and there is also `lbr` that doesn't require special flags.

Reply:
From documentation https://man7.org/linux/man-pages/man1/perf-record.1.html, `lbr` has some limitations, so probably it does not make sense to write in CMakeLists.
```
Using the "lbr" method doesn't require any compiler options. It
will produce call graphs from the hardware LBR registers. The
main limitation is that it is only available on new Intel platforms, such as Haswell.
It can only get user call chain. It doesn't work with branch stack sampling at the same time.
```